G3D10/samples/starter/source/App.cpp

```diff
@@ -81,9 +80,16 @@ void App::onInit() {
     loadScene(
 
 #       ifndef G3D_DEBUG
-            "G3D Sponza"
+        //"G3D Sponza"
+        //"G3D Veach Door"
+        "G3D Simple Cornell Box (Mirror)"
```

Comment:
Are these changes to the default scenes for the starter project intended to make it back to the G3D sample, or is this just for testing the current pull request?